hack/dev/Dockerfile.okteto

```diff
@@ -8,6 +8,10 @@ RUN curl -k https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
   && pip install s3cmd \
   && rm -rf /var/lib/apt/lists/*
 
+RUN apt-get update -y \
+  && apt-get install -y libdevmapper-dev golang-github-proglottis-gpgme-dev libbtrfs-dev \
```

Comment:
What are these additional dependencies used for?

Reply:
These packages are needed to build the kotsadm projects, since some of the modules are c/c++ compiled libs are dependent on these packages.